### PR TITLE
Add regex for various API keys

### LINKED ISSUE #42: API keys Regex
Love the tool BTW,

https://www.redpacketsecurity.com/secretfinder-a-python-script-for-find-sensitive-data-apikeys-accesstoken-jwt-and-search-anything-on-javascript-files/?utm_source=rss&utm_medium=rss&utm_campaign=secretfinder-a-python-script-for-find-sensitive-data-apikeys-accesstoken-jwt-and-search-anything-on-javascript-files

This has some apikey regex's which would be useful.

### PR DESCRIPTION
**Pull request by ryansonshine**
This change adds checks for API keys from:
* Google
* AWS
* Mailgun
* Twilio
* Square
* Stripe
* GitHub
* Slack

The API keys in the tests have either been created by myself and then
invalidated or found on pastebin and confirmed to no longer be active.

Closes #42

### CODE REVIEW

File: pywhat/Data/regex.json [hunk 1]
```diff
@@ -163,6 +163,171 @@
...
+         "API Keys",
+         "Credentials",
+         "AWS"
+      ]
+   },
+    {
+      "Name": "Mailgun API Key",
+      "Regex": "key-[0-9a-zA-Z]{32}",
+      "plural_name": false,
+      "Description": "[#CAE4F1][link=https://documentation.mailgun.com/en/latest/api-intro.html#introduction]API Key for Mailgun[/link][/#CAE4F1]",
```
Comment (bee-san) [inline pywhat/Data/regex.json:222]: Hey! What do you think about getting rid of these descriptions? I believe they're the same as the name, just with a link attached? 😄

**Comment by bee-san** [comment]
Thanks so much for contributing!!!!

File: pywhat/Data/regex.json (hunk 1 above)
Comment (ryansonshine) [reply pywhat/Data/regex.json:222]: Yeah that seems reasonable!

**Review by bee-san — APPROVED** [review]

**Comment by bee-san** [comment]
thank you!!!

**Pull request merged**